tool/requote.js

```diff
@@ -51,9 +51,10 @@ function findSpanContaining(index, spansBySentencePos) {
     return false;
 }
 
+// check normalized version of tokens to handle accent marks/ grapheme properly
 function substringIsFound(sequence, substring, i) {
     for (let j = 0; j < substring.length; j++) {
-        if (sequence[i+j] !== substring[j])
+        if (sequence[i+j].normalize() !== substring[j].normalize())
```

Comment:
This code is run on tokenized examples (sentence/programs) so everything is NFKD already. Why is this needed?